pmultiqc/modules/quantms/quantms.py

```
@@ -80,167 +80,201 @@ def __init__(self):
+                self.enable_exp = True
+            if self.enable_exp == False:
+                for f in self.find_log_files("quantms/sdrf", filecontents=False):
+                    self.sdrf = os.path.join(f["root"], f['fn'])
+                    OpenMS().openms_convert(self.sdrf, config.kwargs['raw'],
+                                            False, True, False, config.kwargs['condition'])
+                    # experimental_design.tsv is the default output name
+                    self.exp_design = os.path.join(f["root"], f['experimental_design.tsv'])
+                    self.enable_sdrf = True
+
+            # TODO in theory the module can work without the design. We just need to remove the according sections!
+            if self.enable_sdrf == False and self.enable_exp == False:
```

Ruff

Avoid equality comparisons to `False`; use `if not self.enable_sdrf:` for false checks

Replace with `not self.enable_sdrf`

(E712)